build.xml

```diff
@@ -245,27 +245,22 @@
         <string>--add-exports java.base/java.lang.ref=ALL-UNNAMED</string>
         <string>--add-exports jdk.unsupported/sun.misc=ALL-UNNAMED</string>
         <string>--add-exports jdk.compiler/com.sun.tools.javac.file=ALL-UNNAMED</string>
+        <string>--add-exports jdk.attach/sun.tools.attach=ALL-UNNAMED</string>
```

Comment:
Why do we need this one? It is apparently not needed by Chronicle Queue?

Reply:
This one was added for the SJK integration with nodetool as part of CASSANDRA-18242, but I added it only to the files and not here back then.